validate/validate.go

```diff
@@ -228,17 +242,17 @@ func (v *Validator) CheckProcess() (msgs []string) {
 
 	process := v.spec.Process
 	if !filepath.IsAbs(process.Cwd) {
-		msgs = append(msgs, fmt.Sprintf("cwd %q is not an absolute path", process.Cwd))
+		errs = multierror.Append(errs, fmt.Errorf("cwd %q is not an absolute path", process.Cwd))
```

Comment:
The MUST for this check is [here][1].

[1]: https://github.com/opencontainers/runtime-spec/blame/v1.0.0/config.md#L155

Reply:
This PR is quite big. I think we could change other errors in the following PRs.